base/operators.jl

```diff
@@ -132,8 +146,8 @@ isequal(x::AbstractFloat, y::Real         ) = (isnan(x) & isnan(y)) | signequal(
 """
     isless(x, y)
 
-Test whether `x` is less than `y`, according to a fixed total order.
-`isless` is not defined on all pairs of values `(x, y)`. However, if it
+Test whether `x` is less than `y`, according to a fixed total order (defined together with
+[`isequal`](@ref)). `isless` is not defined on all pairs of values `(x, y)`. However, if it
```

Comment:
this addition feels redundant to me with the first bullet point

Reply:
I wanted to sneak in a `@ref`, mostly. I could revert it?

Reply:
You could add the `@ref` to the first bullet point instead.

Reply:
But I need to ref just `isequal` don't I? I can't add a link to inside a code snippet/block - I need to wrap an entire snippet. Or is the `@ref` thing actually smart enough to know I'm referring to just the function, like `?` at the REPL?

Reply:
I think I got it - https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#Named-doc-@refs-1